packages/nocodb/src/lib/db/sql-data-mapper/lib/sql/functionMappings/mysql.ts

```diff
@@ -57,6 +58,17 @@ const mysql2 = {
       END${colAlias}`
     );
   },
+  DATETIME_DIFF: ({ fn, knex, pt, colAlias }: MapFnArgs) => {
+    console.log(pt);
+    const date1 = fn(pt.arguments[0]);
+    const date2 = fn(pt.arguments[1]);
+    const rawUnit = pt.arguments[2]
+      ? fn(pt.arguments[2]).bindings[0]
+      : 'minutes';
+    const unit = convertUnits(rawUnit, 'mysql');
+
+    return knex.raw(`TIMESTAMPDIFF('${unit}', ${date2}, ${date1}) ${colAlias}`);
```

Comment:
I believe this won't work. The unit shouldn't be wrapped by quotes.

```
select TIMESTAMPDIFF('YEAR', "2022-01-01", "2021-01-01");
```

which results in

```
SQL Error [1064] [42000]: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''YEAR', "2022-01-01", "2021-01-01")
LIMIT 0, 200' at line 1
```

and

```
select TIMESTAMPDIFF(YEAR, "2022-01-01", "2021-01-01");
```

which results in

```
-1
```